db/connection.go

```diff
@@ -80,14 +78,23 @@ func (c *Connection) Migrate(ctx context.Context) {
 		log.Println("Table does not exist")
 		// insert data
 		result := c.DB.MustExecContext(ctx, videoInsert)
-		fmt.Println(result)
+		count, _ := result.RowsAffected()
+		if count > 0 {
+			log.Println("Inserted data")
+		}
 	} else {
-		row.Scan(&count)
+		err := row.Scan(&count)
+		if err != nil {
+			log.Println(err)
+		}
 		if count == 0 {
```

Comment:
While not a part o this PR I think you could probably cleanup this if/else logic to eliminate the else and improve readability.

Reply:
booo...  
Do be fair, this is super hacky and I want to refactor the entire migration sequence in the future

Reply:
the whole things needs to be gutted and redone